README.md

```diff
@@ -109,15 +112,15 @@ Once you have an instance of the Sidetree class with the suitable adapters, you
 #### Create a DID
 
 ```js
-const { Sidetree, MnemonicKeySystem } = require('@transmute/element-lib');
+const { Sidetree, MnemonicKeySystem } = require("@transmute/element-lib");
 
 // Instantiate the Sidetree class
 const element = new Sidetree(/* See previous section for how to initialize the Sidetree class*/);
 
 // Generate a simple did document model
 const mks = new MnemonicKeySystem(MnemonicKeySystem.generateMnemonic());
-const primaryKey = await mks.getKeyForPurpose('primary', 0);
-const recoveryKey = await mks.getKeyForPurpose('recovery', 0);
+const primaryKey = await mks.getKeyForPurpose("primary", 0);
```

Comment:
I think our convention is to use single quotes in this project

Reply:
This is a readme file... and it was corrected based of the README formatting process.

Reply:
makes me wonder why the format would be different, but not enough to want to fix it :)

Reply:
Ok!